src/i18n/General.js

```diff
@@ -195,4 +195,36 @@ export default {
     zh: '所选时间不能早于当前时间',
     en: 'The selected time cannot be earlier than the current time',
   },
+  enableFlapping: {
+    zh: '启用抖动检测',
+    en: 'Enable Flapping Detection',
+  },
+  enableDesc: {
+    zh: '启用连接抖动的检测功能',
+    en: 'Enable connection flapping detection feature',
+  },
+  windowTime: {
+    zh: '检测时间窗口',
+    en: 'Detection Window Time',
+  },
+  windowTimeDesc: {
+    zh: '进行连接抖动检测时的时间窗口长度',
+    en: 'The length of time window for flapping detection',
+  },
+  maxCount: {
+    zh: '最大断连次数',
+    en: 'Max Disconnection Count',
+  },
+  maxCountDesc: {
+    zh: '在一个检测时间窗口内，MQTT 连接客户端被允许的最大断连次数',
+    en: 'Maximum disconnects allowed for a MQTT client within a detection window time',
```

Comment:
Maximum allowed number of disconnections for an MQTT client within a specified detection time window

Reply:
Thanks. Fixed on: https://github.com/emqx/emqx-dashboard5/pull/1572